cli/generators/crud/index.js

```diff
@@ -7,6 +7,13 @@ module.exports = class extends Generator {
     super(args, opts);
 
     this.argument('name', { type: String, required: true });
+
+    this.option('api', {
+      desc: 'Does not include API related files',
```

Comment:
Is this an indicator of what the option does? If so, would it be better that it indicates it in imperative? ("Do not include API related files")